### DISCUSSION TITLE
What kind of projects does the license allow?

### DISCUSSION THREAD

**Discussion by MatMiler** [microsoft/fluentui#18407] [General] [answered]
The microsoft/fluentui project itself has a MIT license, which (at least as far as I understand) allows just about anything. But the assets' (fonts & icons) license limits the use of assets to applications which integrate or interact with some Microsoft service or application, presumably in a significant enough way to require consistent UI and branding.

To be honest, I can't work out what exactly do the two licenses mean together. Is using the Fluent UI for web likewise limited to only integrating/interacting with Microsoft's products? Or is the assets' license satisfied merely because the assets are already used with (Microsoft's) Fluent UI toolkit? Or can different fonts and icons be used to avoid the issue?

So, on what kind of projects can this be used? Must they visibly feature one or more of Microsoft's services or applications? Or can it also be used on open-source projects? Possibly any kind of web application?

**Comment by jtbandes** [comment, marked as answer]
If you don't need the branded icons, you can use the SVG icons from `@fluentui/react-icons-mdl2`, which is MIT-licensed.

Read more at https://developer.microsoft.com/en-us/fluentui#/styles/web/icons

**Reply by MatMiler** [reply]
Thanks.
